### PR TITLE
add: landing-page

### PR DESCRIPTION
**Pull request by devsurajrai**
Added the landing page for my component library. Please review. Suggestions are most welcome.

### CODE REVIEW

File: components/docs/docs.css [hunk 1]
```diff
@@ -0,0 +1,102 @@
+/* navbar  */
+.docs {
+  background-color: var(--secondary);
+}
+/* sidebar  */
+.components-container {
+  display: grid;
+  /* grid-auto-flow: column; */
+  grid-template-columns: 1fr 5fr;
```
Comment (vishal804) [inline components/docs/docs.css:9]: Try to Remove commented code If not needed

Comment (vishal804) [reply components/docs/docs.css:9]: Everything Else Looks Good

Comment (devsurajrai) [reply components/docs/docs.css:9]: > Try to Remove commented code If not needed

Thanks, Will do that.

File: components/docs/docs.css [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+.component-heading::after {
+  content: "";
+  border-right: 4px solid var(--primary);
+  margin: 1rem;
+}
+.component-heading::before {
+  content: "";
+  border-right: 4px solid var(--primary);
+  margin: 1rem;
+}
```
Comment (ApurvaSawant11) [inline components/docs/docs.css:40]: If the content is the same for both of them then use a comma separator.  
`.component-heading::before, .component-heading::after {  .......  }`

**Review by ApurvaSawant11 — APPROVED** [review]
Looks good to me!

File: components/docs/docs.css (hunk 2 above)
Comment (devsurajrai) [reply components/docs/docs.css:40]: Thanks

File: .vscode/settings.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
```
Comment (nshanbhag20001412) [inline .vscode/settings.json:2]: this can be moved to .gitignore

File: components/alert/alert.css [hunk 4]
```diff
@@ -12,46 +12,38 @@
...
 }
 .alert--primary {
-  background-color: #fda889;
+  background-color: #abfaf2;
   color: var(--primary);
-  border-color: #fc8559;
 }
 .alert--secondary {
-  background-color: #9fbcd9;
+  background-color: #8f86cc;
```
Comment (nshanbhag20001412) [inline components/alert/alert.css:19]: we can use color variable here

Comment (devsurajrai) [reply components/alert/alert.css:19]: thanks. will do that

**Pull request merged**